src/components/KeyboardSignature.tsx

```diff
@@ -305,16 +309,31 @@ export const KeyboardSignature = () => {
                   <button
                     key={type}
                     onClick={() => setCurveType(type)}
-                    className={`px-3 py-1 text-xs rounded-full transition-all duration-150 ease-out cursor-pointer border ${
-                      curveType === type
-                        ? "bg-white text-black font-medium border-white"
-                        : "bg-neutral-900/50 text-neutral-400 hover:bg-neutral-800/50 hover:text-neutral-200 border-neutral-800"
-                    }`}
+                    className={`px-3 py-1 text-xs rounded-full transition-all duration-150 ease-out cursor-pointer border ${curveType === type
+                      ? "bg-white text-black font-medium border-white"
+                      : "bg-neutral-900/50 text-neutral-400 hover:bg-neutral-800/50 hover:text-neutral-200 border-neutral-800"
+                      }`}
                   >
                     {type.replace("-", " ")}
                   </button>
                 ))}
               </div>
+
+              {/* Numbers Toggle */}
+              <p className="text-neutral-300 text-sm font-medium mr-8 mt-1">
```

Comment:
You can remove the `mt-1` - since the toggle is the same height as the line it's already centered without margin